sample.env

```diff
@@ -4,43 +4,43 @@
 
 LOCAL_GHOST_API_URL=api_url_from_ghost_dashboard
 LOCAL_GHOST_CONTENT_API_KEY=api_key_from_ghost_dashboard
-LOCAL_GHOST_API_VERSION=v3
+LOCAL_GHOST_API_VERSION=v3.42
 
 ENGLISH_GHOST_API_URL=api_url_from_ghost_dashboard
 ENGLISH_GHOST_CONTENT_API_KEY=api_key_from_ghost_dashboard
-ENGLISH_GHOST_API_VERSION=v3
+ENGLISH_GHOST_API_VERSION=v3.42
```

Comment:
Is this the correct version? I'm seeing 3.39.0 on `/news/ghost`

Reply:
Hey @nhcarrigan, thanks for checking this out. Sorry, I could have included more details in my description above.

No, the versions of Ghost we're using are all over the place. The ones listed here are the most up-to-date versions of 3 and 2.

In my test builds, using the latest versions seemed to work. The minor version didn't seem to make a difference, at least as far as I could tell. But in the future something could change on Ghost's end that causes builds to fail if the major/minor versions don't match exactly.

My current thinking is that we can just list the latest versions of v3 and v2 in the `sample.env` file, and have the exact major/minor versions on Orbit, Azure, and in the LastPass notes where all the env vars are listed. I figure that any of us who work with this repo at least have access to LastPass, or can reach out on Signal for everything.

I don't think there's a security issue with listing the exact major/minor versions in the `sample.env` file. But it might be better to obscure those in case there are any issues I missed in Ghost's release notes.

Reply:
Ah, or what we could do is change all of these values to placeholders like `v{major}.{minor}` so it's more clear that they'll need to be replaced.

How does that sound?

Reply:
I think that's a good approach, yeah

Reply:
Hey @nhcarrigan, thanks for your patience and confirming that.

I'll go ahead and make those updates now :slightly_smiling_face: